lib/csvlint/validate.rb

```diff
@@ -331,10 +327,6 @@ def line_breaks
       end
     end
 
-    def row_count
```

Comment:
I'd prefer if we could keep the row_count variable. It's being used in some tests. Could we just keep a variable which we add one to every time we process a row?

Reply:
Done